packages/data-models/tests/treebank_data_item.test.js

```diff
@@ -35,69 +39,72 @@ describe('treebank_data_item.test.js', () => {
 
   it('2 TreebankDataItem - fail on reference missing', () => {
     const srcElem = document.createElement('div')
-    srcElem.setAttribute('data-alpheios_tb_app','perseids-treebank-template')
+    srcElem.setAttribute('data-alpheios_tb_app', 'perseids-treebank-template')
     srcElem.setAttribute('data-alpheios_tb_app_version', '1')
     srcElem.setAttribute('data-alpheios_tb_app_url', 'http://example.org/DOC/SENTENCE/WORD')
     document.body.appendChild(srcElem)
     const wdElem = document.createElement('span')
-    document.body.appendChild(wdElem)
+    wdElem.setAttribute('data-alpheios_tb_doc', 'document')
+    srcElem.appendChild(wdElem)
     expect.assertions(1)
     try {
       const item = new TreebankDataItem(wdElem)
-    } catch(error) {
-      expect(error).toHaveProperty('message','Cannot find an element with the data-alpheios_tb_ref data attribute among the ancestors')
+    } catch (error) {
+      expect(error).toHaveProperty('message', 'An element does not have data-alpheios_tb_ref, data-alpheios_tb_word or data-alpheios_tb_sent attributes')
     }
   })
 
   it('3 TreebankDataItem - fail on invalid reference', () => {
     const srcElem = document.createElement('div')
-    srcElem.setAttribute('data-alpheios_tb_app','perseids-treebank-template')
+    srcElem.setAttribute('data-alpheios_tb_app', 'perseids-treebank-template')
     srcElem.setAttribute('data-alpheios_tb_app_version', '1')
     srcElem.setAttribute('data-alpheios_tb_app_url', 'http://example.org/DOC/SENTENCE/WORD')
     document.body.appendChild(srcElem)
     const wdElem = document.createElement('span')
     wdElem.setAttribute('data-alpheios_tb_ref', 'treebank')
-    document.body.appendChild(wdElem)
+    wdElem.setAttribute('data-alpheios_tb_doc', 'document')
```

Comment:
why is tb_doc added for this test?

Reply:
The `TreebankDataItem` will throw an exception if selection has no `data-alpheios_tb_doc` in its surrounding. So in order to create a `TreebankDataItem`, we need to have `data-alpheios_tb_doc` somewhere, Is that correct?

Reply:
Oh, just realized this is a `data-alpheios_tb_ref` here, where the doc ID is already in a reference. Will double check that tomorrow.